projects/kit/components/input-range/input-range.template.html

```diff
@@ -37,42 +37,32 @@
         </div>
     </tui-input-number>
 
-    <div class="t-right">
-        <tui-input-number
-            tuiTextfieldAppearance="none"
-            automation-id="tui-input-range__right-input"
-            class="t-text-input"
-            [min]="value[0]"
-            [max]="max"
-            [precision]="precision"
-            [decimal]="decimal"
-            [disabled]="computedDisabled"
-            [readOnly]="readOnly"
-            [ngModel]="value[1]"
-            (ngModelChange)="onInputRight($event)"
-            (focusedChange)="onTextInputFocused($event, true)"
-            (keydown.arrowUp)="changeByStep($event, [0, 1])"
-            (keydown.arrowDown)="changeByStep($event, [0, -1])"
-        >
-            <div
-                *ngIf="showRightValueContent"
-                ngProjectAs="tuiContent"
-            >
-                <ng-container *polymorpheusOutlet="rightValueContent as text; context: {$implicit: value[1]}">
-                    {{ text }}
-                </ng-container>
-            </div>
-        </tui-input-number>
-
-        <!-- TODO replace by postfix of the right InputNumber (after fix https://github.com/Tinkoff/taiga-ui/issues/1193) -->
-        <span
-            *ngIf="!showRightValueContent && pluralize"
-            automation-id="tui-input-range__pluralize-right"
-            class="t-pluralize-right"
+    <tui-input-number
+        tuiTextfieldAppearance="none"
+        automation-id="tui-input-range__right-input"
+        class="t-right"
+        [min]="value[0]"
+        [max]="max"
+        [precision]="precision"
+        [decimal]="decimal"
+        [disabled]="computedDisabled"
+        [tuiTextfieldPostfix]="pluralize && !showRightValueContent ? (value[1] | i18nPlural : pluralize) : ''"
```

Comment:
Don't merge until this issue will be closed

* https://github.com/Tinkoff/maskito/issues/379